explainer.md

```diff
@@ -17,86 +15,140 @@ The two drafts apply to both TLS server certificates, where the server authentic
-* Minimize bandwidth cost to the TLS handshake.
+## Design Goals
 
-Matching the standard pattern for other TLS parameters, we target a deployment model where subscribers can configure multiple TLS certificates, with TLS software automatically sending the right one on each connection. To aid in this, CAs can transparently provision subscribers with multiple TLS certificates with ACME or another automated protocol.
+In solving this problem, there were a few notable design goals to meet:
 
-We discuss these motivations in more depth below.
+* Enable PKIs to evolve as needed for user security, in a timely manner and without conflicting with availability. We are particularly focused on PKIs that share characteristics with the existing Web PKI.
+* Minimize bandwidth cost in the TLS handshake. With the post-quantum transition, cryptographic material will become [much more expensive](https://dadrian.io/blog/posts/pqc-signatures-2024/) to send.
+* Minimize burden to server operators, particularly avoiding ongoing manual work. Most deployments should leave ongoing decisions to TLS software, ACME software, and ACME server behavior.
 
-### Why Multiple Certificates?
+We discuss these in more detail below:
 
-PKIs need to evolve over time to meet user security needs. For example:
+### Timely PKI Evolution
 
-* CAs that add net value to the ecosystem may be added to relying parties
-* Long-lived CA keys should be rotated to reduce risk
-* Compromised or untrustworthy CAs are removed from relying parties
+If a CA is compromised or otherwise considered untrustworthy, mitigating the TLS interception attacks requires that the client no longer trust the CA. This means we should minimize delays in completing PKI transitions at the client, as that delay translates to time during which users remain at risk.
 
-These kinds of changes inherently create divergence, if for no other reason than
-out-of-date clients in the ecosystem. These clients could range from unupdatable
-TV set-top boxes to some IoT device to an individual user's browser that could
-not communicate with its update service.
+In other transitions, reducing delays is also valuable. If trustworthy CA is considered to provide net value to the ecosystem and added, a timely transition allows servers to make use of the CA and benefit.
 
-Today, these old clients limit security improvements for other, unrelated
-clients. Consider a TLS client making some trust change for user security. For
-availability, TLS servers must have some way to satisfy it. Some server,
-however, may also support an older client. If the server uses a single
-certificate, that certificate is limited to the intersection of both clients.
+Negotiation mechanisms naturally achieve this, following the standard TLS pattern, by letting the server serve any collection of certificates it needs to maintain availability. The server is not constrained by availability of CA cross-signs, which carry signficant risks for all parties in the PKI.
```

Comment:
What risks? (this is the question that will be asked, do we leave it unanswered here?)

Reply:
It feels like we can TODO a link to separate text that explains this later.